src/wallet/rpcwallet.cpp

```diff
@@ -1403,10 +1434,15 @@ UniValue listtransactions(const JSONRPCRequest& request)
                     {"label", RPCArg::Type::STR, /* opt */ true, /* default_val */ "null", "If set, should be a valid label name to return only incoming transactions\n"
             "              with the specified label, or \"*\" to disable filtering and return all transactions."},
                     {"count", RPCArg::Type::NUM, /* opt */ true, /* default_val */ "10", "The number of transactions to return"},
-                    {"skip", RPCArg::Type::NUM, /* opt */ true, /* default_val */ "0", "The number of transactions to skip"},
+                    {"skip", RPCArg::Type::NUM, /* opt */ true, /* default_val */ "0", "The number of transactions to skip, In case of flags contain nextpagepointer, fThe value of it is a string given by listtransactions."},
                     {"include_watchonly", RPCArg::Type::BOOL, /* opt */ true, /* default_val */ "false", "Include transactions to watch-only addresses (see 'importaddress')"},
+                    {"flags", RPCArg::Type::STR, /* opt */ true, /* default_val */ "", "list of flags separated by | which they are [ nextpagepointer ]"},
```

Comment:
nit: I think we either use a json object called `options` or a named argument directly. A specially serialized `flags` seems inappropriate given that we have all json types and named arguments available.